frameworks/helloworld/universe/config.json

```diff
@@ -112,6 +113,11 @@
           "description": "Allow region awareness",
           "type": "boolean",
           "default": true
+        },
+        "custom_service_tld": {
```

Comment:
Would `custom_top_level_domain` not be better here? Duplicating `service` may be unnecessary?

Reply:
I was matching it to the name of the envvar used

Reply:
We could have the envvar be `SERVICE_CUSTOM_TLD` in the same way that we use `SECURITY_KERBEROS_ENABLED` to also (partially) encode the json heirarchy.

Reply:
FWIW, I still think `service.custom_tld` is better than `service.custom_service_tld`, but I'm not adamant about it.